src/syscall.h

```diff
@@ -8,14 +8,36 @@ See the file LICENSE for details.
 #define SYSCALL_H
 
 #include "kerneltypes.h"
+#include "graphics.h"
```

Comment:
Is this for graphics_color? Then we should move the declaration of the struct to global_types.h as well. I think we want to minimize the amount of files syscall.h is importing.